### ISSUE TITLE
[BUG 🐞] Footer comes when scrolled in mobile view

### ISSUE THREAD

**Issue by warmachine028** [warmachine028/nike#84]
### Description

- Scrolling down the page brings the footer below the menu
- This must be fixed and the footer must be behind the menu always.

### Screenshots

![Screenshot_2023-10-02-20-20-34-045_com android chrome](https://github.com/warmachine028/nike/assets/75939390/a22e4b90-8c2c-45a4-82df-5e300e751649)

## Happy contributing. 💐
Star my other Repositories [here](https://github.com/warmachine028)

**Comment by warmachine028** [comment]
Assigning you to this issue.

**Comment by warmachine028** [comment]
Have you started working on this first?

**Comment by a-s-t-e-y-a** [comment]
@warmachine028 Check my PR  i solved the issue

**Comment by warmachine028** [comment]
@a-s-t-e-y-a next time please get assigned to an issue first.

I'll wait for @rakimsth to respond for some time, if he doesn't I'll review your PR 
- Then Request if changes are needed.
- then add hacktoberfest tags and accept your contribution

**Comment by a-s-t-e-y-a** [comment]
@warmachine028 
Okay I'll take care of this from next time 
But you assigned this issue more than 10 hours ago so I thought why wouldn't  I'll fix this

**Comment by warmachine028** [comment]
> @warmachine028 
> Okay I'll take care of this from next time 
> But you assigned this issue more than 10 hours ago so I thought why wouldn't  I'll fix this

Yes you have a valid concern regarding that, i totally understand

**Comment by warmachine028** [comment]
See #76 if you can solve it or on my other projects there are pending issues too

**Comment by warmachine028** [comment]
Have a look at [this](https://github.com/warmachine028/memories/issues/87)
You have worked on a similar issue before

**Comment by warmachine028** [comment]
@rakimsth I am assigning @a-s-t-e-y-a to this issue, please work on the other issue you are assigned already.
